maskit/masked_parameters.py

```diff
@@ -103,42 +130,63 @@ def _perturb_wires(self, amount: int = None, mode: PerturbationMode = Perturbati
         if len(indices) == 0:
             return
         if random:
-            layer_indices = [slice(None, None, None),
-                             np.random.choice(indices, min(count, len(indices)),
-                                              replace=False)]
+            layer_indices = [
+                slice(None, None, None),
+                np.random.choice(indices, min(count, len(indices)), replace=False),
+            ]
         else:
             layer_indices = indices[:count]
         self._mask[layer_indices] = ~self._mask[layer_indices]
 
-    def _perturb_random(self, amount: int = None, mode: PerturbationMode = PerturbationMode.INVERT, random: bool = True):
-        count = abs(amount) if amount is not None else rand.randrange(0, self._params.size)
+    def _perturb_random(
+        self,
+        amount: int = None,
+        mode: PerturbationMode = PerturbationMode.INVERT,
+        random: bool = True,
+    ):
+        count = (
+            abs(amount) if amount is not None else rand.randrange(0, self._params.size)
+        )
         if mode == PerturbationMode.REMOVE:
             indices = np.argwhere(self._mask)
             if len(indices) == 0:
                 return
             if random:
-                random_indices = tuple(zip(*indices[
-                    np.random.choice(len(indices), min(count, len(indices)),
-                                         replace=False)]))
+                random_indices = tuple(
+                    zip(
+                        *indices[
+                            np.random.choice(
+                                len(indices), min(count, len(indices)), replace=False
+                            )
+                        ]
+                    )
+                )
             else:
                 random_indices = tuple(zip(*indices[:count]))
         elif mode == PerturbationMode.ADD:
             indices = np.argwhere(~self._mask)
             if len(indices) == 0:
                 return
             if random:
-                random_indices = tuple(zip(*indices[
-                    np.random.choice(len(indices), min(count, len(indices)),
-                                     replace=False)]))
+                random_indices = tuple(
+                    zip(
+                        *indices[
+                            np.random.choice(
+                                len(indices), min(count, len(indices)), replace=False
+                            )
+                        ]
+                    )
+                )
             else:
                 random_indices = tuple(zip(*indices[:count]))
         else:
```

Comment:
If this isn't a catch-all branch, consider to check for an enum value here and have a final ``else: raise ValueError(...)``.